.github/workflows/build.yml

```diff
@@ -224,6 +224,60 @@ jobs:
           name: freebsd-${{ matrix.processor }}-ds2
           path: ${{ github.workspace }}/BinaryCache/ds2.tar
 
+  tizen:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - processor: arm
+            cmake_args: |
+              -D CMAKE_SYSTEM_PROCESSOR=arm               \
+              -D CMAKE_C_COMPILER=arm-linux-gnueabi-gcc   \
+              -D CMAKE_CXX_COMPILER=arm-linux-gnueabi-g++
+            packages: gcc-arm-linux-gnueabi g++-arm-linux-gnueabi
+
+          - processor: x86
+            cmake_args: |
+              -D CMAKE_SYSTEM_PROCESSOR=i686  \
+              -D CMAKE_C_COMPILER=gcc         \
+              -D CMAKE_CXX_COMPILER=g++       \
+              -D CMAKE_C_FLAGS=-m32           \
+              -D CMAKE_CXX_FLAGS=-m32
+            packages: gcc-multilib g++-multilib
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          path: ${{ github.workspace }}/SourceCache/ds2
+
+      - name: Install compilers
+        run: |
+          sudo apt-get install -qq --no-install-recommends ${{ matrix.packages }}
```

Comment:
Hmm, we don't have the Tizen sysroot installed that we should be building against. This is effectively a GCC build for Ubuntu, which I wouldn't be against, but this doesn't cover Tizen unfortunately.

Reply:
I thought using the ARM cross compiler would be good enough, but yeah that make sense. I will install the Tizen SDK and see about using that.